requirements/edx/pre.txt

```diff
@@ -4,6 +4,9 @@
 #   * @mollydb - to check licensing
 #   * One of @e0d, @jarv, or @feanil - to check system requirements
 
+# Use a modern setuptools instead of distribute
+setuptools==9.1
```

Comment:
NLTK is now at 2.0.5 for edx-platform and edx-ora2. Can we make this PR target the latest stable version of setuptools?